astropy/stats/tests/test_funcs.py

```diff
@@ -258,7 +258,7 @@ def test_biweight_midcovariance():
     assert_allclose(std, [ 2.6,  2.1,  1.6,  1.5,  1.9])
     # Test 3: Ensure matrix is symmetric
     rng = np.random.RandomState(1)
-    d = np.array([rng.gamma(2,2,500) for i in range(3)])
+    d = np.array([rng.gamma(2, 2, 500) for i in range(3)])
     cov = funcs.biweight_midcovariance(d)
     assert_allequal(cov, cov.T)
```

Comment:
``assert_allequal`` should be ``assert_equal``.
I suggest that you run the test suite locally before pushing your commits ;-)